data/moves.js

```diff
@@ -4116,22 +4116,11 @@ let BattleMovedex = {
 		pp: 10,
 		priority: 0,
 		flags: {protect: 1, mirror: 1},
-		multihit: 2,
 		secondary: null,
 		onModifyMove(move, pokemon, target) {
-			if (target.side.active.length === 2) {
-				move.multihit = 1;
-				move.spreadModifier = 1;
-				move.target = "allAdjacentFoes";
-				for (const currentTarget of target.side.active) {
-					if (currentTarget.volatiles['protect'] || !currentTarget.runImmunity('Dragon') || currentTarget.fainted) {
-						move.multihit = 2;
-						break;
-					}
-				}
-			}
+			if (target.side.active.length === 2) move.spreadModifier = 1;
```

Comment:
```suggestion
			move.spreadModifier = 1;
```

Reply:
Or you could just put the `spreadModifier` directly in the move data?

Reply:
I think this probably shouldn't be considered a spread move at all, but a multihit move that smart-targets every hit.